templates/openstack/developer.html

```diff
@@ -1,14 +1,16 @@
 {% extends "openstack/_base_openstack.html" %}
 {% block title %}Developer | OpenStack | Ubuntu{% endblock %}
-{% block meta_description %}Want to develop on OpenStack. Use conjure-up to install it on your laptop with LXD.{% endblock %}
+{% block meta_description %}Step-by-step instructions to deploy a full OpenStack cluster on your workstation using LXD machine containers for service isolation.{% endblock %}
 
 {% block content %}
 <section class="p-strip--light is-deep is-bordered">
   <div class="row">
     <div class="col-6">
       <div>
-        <h1>Developer OpenStack</h1>
-        <p>Install OpenStack on a single machine with Conjure-up. It deploys a real-world OpenStack on a single machine using LXD, that perfectly models a production cluster on real machines.</p>
+        <h1>Single-node Developer OpenStack cluster</h1>
+        <p>Install a full OpenStack cluster on a single machine.This guide uses <a class="p-link--external" href="https://linuxcontainers.org/">LXD</a>, the Linux Container Daemon, to create multiple machine containers for the various OpenStack services, mimicking a real production distributed Openstack cluster.</p>
+        <p>Follow these step-by-step instructions.</p>
+        <p><strong>conjure-up</strong> requires a minimum version of LXD of <strong>3.0.0</strong>. Additionally, LXD should be configured prior to running.</p>
       </div>
     </div>
     <div class="col-6">
```

Comment:
The 'u-align--center u-vertically-center u-hide--small' needs to be on the div above, not the image

Reply:
the /openstack/install page likely needs to match this